src/runtime/graph/graph_runtime.cc

```diff
@@ -429,7 +449,10 @@ void GraphRuntime::LoadParams(dmlc::Stream* strm) {
     CHECK_GE(in_idx, 0) << "Found param for non-existent input: " << names[i];
     uint32_t eid = this->entry_id(input_nodes_[in_idx], 0);
     CHECK_LT(eid, data_entry_.size());
-    LoadDLTensor(strm, &data_entry_[eid]);
+
+    NDArray temp;
```

Comment:
Add a comment on why a temp Tensor is necessary.

// The data_entry is allocated on device, NDArray.load always load the array into CPU